### ISSUE TITLE
impostare lista punti critici emersi da POC Normalizzazione Cromatica

### ISSUE THREAD

**Issue by lmusso-dermaself** [dermaself/new-skincare-advisor-web#48]

**Comment by 4lessi4** [comment]
Buongiorno Lorenzo,

sono riuscita a raccogliere il feedback dai ragazzi del Tech Team in merito alle principali criticità riscontrate.
Come promesso, ti allego la presentazione del POC aggiornata, che include anche una slide dedicata ai punti critici.

[POC Dermaself.pdf](https://github.com/user-attachments/files/22717975/POC.Dermaself.pdf)

Per quanto riguarda la Normalizzazione Cromatica i punti critici emersi sono questi:

- **Potenza limitata del dispositivo per l’inferenza del modello** (modelli idealmente inferiori a 1 Mb): i dispositivi mobili presentano vincoli stringenti in termini di memoria e capacità di calcolo, pertanto il modello deve rimanere leggero per garantire un funzionamento fluido senza un uso eccessivo delle risorse.

- **Acquisizione solo in formato JPEG**: poiché la cattura e il caricamento delle immagini sono limitati al formato JPEG.

- **Robustezza dell’acquisizione rispetto a immagini inutilizzabili, senza penalizzare l’esperienza utente**: il sistema deve essere in grado di rilevare e gestire immagini di scarsa qualità o difettose in modo efficiente, evitando che l’utente debba ripetere lo scatto o andare incontro ad errori.

- **Efficienza temporale della pipeline di elaborazione (meno di 10 secondi complessivi; normalizzazione e acquisizione entro 3–4 secondi, escluso il trasferimento di rete)**.

- **Per il modello fine-tuned: dati JPEG 1024x1024, compatibilità con tflite (eventualmente convertendo da torch o tensorflow)**.

- **Definizione di metriche di valutazione quantificabili per la verifica oggettiva della qualità, insieme a esempi concreti di casi Dermaself pre/post normalizzazione**.

- **Adozione delle soluzioni su entrambe le piattaforme iOS e Android** per garantire usabilità universale e ampia copertura di mercato.

**Comment by lmusso-dermaself** [comment]
ciao @4lessi4 , riguardo ai punti sopra , il team è già andato avanti nelle investigazioni? puoi creare task di investigazione e poi quelli di sviluppo ed assegnarli al team.

per quanto riguarda l'adozione delle soluzioni dobbiamo adattarle al browser quindi va migrato e integrato il POC dentor la nostra webapp che si trova in questo repo e branch https://github.com/dermaself/new-skincare-advisor-web/tree/develop-neutral-demo

puoi assegnare i task in base alle criticità descritte as Adriano e dirgli di farsi un nuovo branch a partire da questo https://github.com/dermaself/new-skincare-advisor-web/tree/develop-neutral-demo e iniziare ad integrare tutto nel flusso della camera già implementato in questa webapp
